Mathlib/Algebra/Order/Monoid/Unbundled/Units.lean

```diff
@@ -13,26 +13,81 @@ import Mathlib.Algebra.Order.Monoid.Unbundled.Basic
 
 variable {M : Type*} [Monoid M] [LE M]
 
-theorem Units.mulLECancellable_val [MulLeftMono M] (a : Mˣ) :
-    MulLECancellable (↑a : M) := fun _ _ h ↦ by
-  simpa using mul_le_mul_left' h ↑a⁻¹
+section MulLeftMono
+variable [MulLeftMono M] (u : Mˣ) {a b c : M} (ha : IsUnit a)
 
-theorem Units.mul_le_mul_left [MulLeftMono M] (a : Mˣ) {b c : M} :
-    a * b ≤ a * c ↔ b ≤ c :=
-  a.mulLECancellable_val.mul_le_mul_iff_left
+namespace Units
```

Comment:
I wouldn't have any lemma about `Units`, only lemmas about `IsUnit`

Reply:
We should have `Units` lemmas for all of the theorems involving `inv`

Reply:
... and none of the other ones

Reply:
I've made it `private`